.travis.yml

```diff
@@ -8,7 +8,6 @@ rvm:
 script:
   - 'bundle exec rails db:migrate RAILS_ENV=test'
   - 'bundle exec rspec spec'
-  - 'bundle exec rubocop'
```

Comment:
Rubocop is disabled in CodeClimate config. We're using our Rubocop rules gem, and CodeClimate can't handle that.